dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM php:7-cli
+
+RUN apt-get update 
+RUN apt-get install -y curl zip
+
+RUN curl -sS https://getcomposer.org/installer | php
```

Comment:
You can actually do `COPY --from=composer /usr/bin/composer /usr/bin/composer` to get the latest binary from the `composer` image instead of running the installer yourself.